src/database/queries/addUser.js

```diff
@@ -0,0 +1,3 @@
+const connction = require('../connection.js');
+const addUser = (user_name,password)=> connction.query(`INSERT INTO users(user_name,password)values($1,$2)`,[user_name,password]);
```

Comment:
@denaHS It's not super important but a few spaces in the query before and after parentheses would help it be a little more readable -
`INSERT INTO users (user_name, password) values ($1, $2)`

On my team, we actually have all of our favorite spacing preferences built into our eslint config 😂